service/qos/qos_configuration_change_subscriber.hh

```diff
@@ -23,15 +23,19 @@
 
 #include "qos_common.hh"
 
+
 namespace qos {
+
+    struct additional_service_level_info {
+    };
     class qos_configuration_change_subscriber {
     public:
         /** This callback is going to be called just before the service level is available **/
-        virtual future<> on_before_service_level_add(sstring name, service_level_options slo) = 0;
+        virtual future<> on_before_service_level_add(sstring name, service_level_options slo, additional_service_level_info sl_info) = 0;
```

Comment:
Let's put name into sl_info (now it's not "additional") to make it less artificial. Otherwise @xemul will send a patch to remove this empty struct.

Reply:
Changed, also changed the commit and PR messages accordingly.